t04_refactored.py

```diff
@@ -171,10 +171,61 @@ def team_9_adv():
 ###################################################################################
 
 def team_10_adv():
-    pass
-    # TODO Add your code here
+"""
+https://docs.google.com/document/d/1p3RkJxJYhLnXdNLjh08KZxLTcqlHuq3bDp9Op5_nHM4/edit
+Eun Sung
+Juno Smith
+:return: none
+"""
+print("A cave guard appears at the cave entrance")
+sleep(delay)
+action = input("Should you: [Talk/Ignore/Punch] him?")
+print("\n")
+
+if action == "Talk":
+    # good choice
+    print("Greeting Traveler, didn't see you there.\nIt's awfully dark in there.\nIf you want to reach the "
+          'treasure,\nyou must answer my riddles and there will be light to guide you.')
+
+elif action == "Ignore":
+    # first neutral choice
+    print("You ignore the cave guard and walk past him.")
+    print("You're still lost and it's still dark. Maybe he knows how to fix that.")
+
+elif action == "Punch":
+    # bad choice
```

Comment:
The original prompt asked for the bad choice to give the user a second chance (e.g., "pick a number").